docs/guides/dev/How-to-build-erda-project.md

```diff
@@ -14,3 +14,21 @@ The following command will build all erda modules. The specific build process ca
 make prepare && make
 ```
 
+## Build a single module locally
+
+The following command will build a specific erda module.
+
+```
+make prepare && MODULE_PATH=<module_name> make build
```

Comment:
Does `build` phrase auto add `prepare` phrase is better?

Reply:
emmmm.. u'r right.